web/taskmaster/src/env.d.ts

```diff
@@ -0,0 +1,16 @@
+// Define the type of the environment variables.
+declare interface Env {
+  readonly NODE_ENV: string;
+  // Replace the following with your own environment variables.
+  // Example: NGX_VERSION: string;
+  readonly PSQL_CONNECTION_STRING: string;
+  readonly SECRET_KEY: string;
+  readonly FB_CLIENT_ID: string;
+  readonly FB_CLIENT_SECRET: string;
+  readonly FB_REDIRECT: string;
```

Comment:
The connection string and secrets should be backend only and I don't think we should load them to the frontend

Reply:
There is a bug when running in local `sdk.js?hash=c611f1a6…4bd57103decf3b35:49 FB.login() called before FB.init().`